Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt

```diff
@@ -391,14 +391,30 @@ or to vertices, or to both :
 Property maps are used to record the computed normals.
 
 
-\subsection NormalsExample Normals Computation Example
+\subsection NormalsExample Normals Computation Examples
+
+Property maps are an API introduced in the boost library, that allows to
+associates values to keys. In the following examples we assocatiate
```

Comment:
there are two typos now
- "associates" should be "associate"
- "assocatiate" should be "associate"